third_party/rust/multihash/v0_16/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License
```

Comment:
there is already a license file in chromium_crates_io for this

Reply:
Removed in 51cb2ab18a26e255dfc52ebe29e19e399b8853d3